src/main/java/ru/yandex/clickhouse/settings/ClickHouseProperties.java

```diff
@@ -203,6 +203,8 @@ public ClickHouseProperties(ClickHouseProperties properties) {
         setSessionCheck(properties.sessionCheck);
         setSessionId(properties.sessionId);
         setSessionTimeout(properties.sessionTimeout);
+        setPreferredBlockSizeBytes(properties.preferredBlockSizeBytes);
```

Comment:
these were not copied from original properties instance, i assume it's an omission, not intentional